.gitlab-ci.yml

```diff
@@ -95,6 +96,7 @@ build-idp-image:
   needs: []
   interruptible: true
   rules:
+    - if: $CI_COMMIT_BRANCH == $CI_DEFAULT_BRANCH
```

Comment:
Is this supposed to be added?

Reply:
I see now that it is so we can run scans on images built from `main`

Reply:
yes, we need it to run the image build against the default branch so we can also run a scan against what is in main